src/Catalyst.Core.Lib/FileSystem/FileSystem.cs

```diff
@@ -0,0 +1,144 @@
+#region LICENSE
+
+/**
+* Copyright (c) 2019 Catalyst Network
+*
+* This file is part of Catalyst.Node <https://github.com/catalyst-network/Catalyst.Node>
+*
+* Catalyst.Node is free software: you can redistribute it and/or modify
+* it under the terms of the GNU General Public License as published by
+* the Free Software Foundation, either version 2 of the License, or
+* (at your option) any later version.
+*
+* Catalyst.Node is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+* GNU General Public License for more details.
+*
+* You should have received a copy of the GNU General Public License
+* along with Catalyst.Node. If not, see <https://www.gnu.org/licenses/>.
+*/
+
+#endregion
+
+using System;
+using System.IO;
+using System.IO.Abstractions;
+using System.Threading.Tasks;
+using Catalyst.Core.Lib.Config;
+using Polly;
+using Polly.Retry;
+using IFileSystem = Catalyst.Abstractions.FileSystem.IFileSystem;
+
+namespace Catalyst.Core.Lib.FileSystem
+{
+    // ReSharper disable once ClassWithVirtualMembersNeverInherited.Global
+    public class FileSystem : System.IO.Abstractions.FileSystem, IFileSystem
+    {
+        // private readonly string _currentDataDirPointer;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/app/catalyst-network/Catalyst.Node/pullRequest?prid=4318201)